gym/envs/algorithmic/copy_.py

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
(I renamed this file because the collision with the python standard library of the same name led to some problems. For example, running `pip` or `nose` in the gym/envs/algorithmic directory would fail with obscure error messages.)